app/index.js

```diff
@@ -49,6 +50,12 @@ module.exports = class GeneratorLicense extends Generator {
       desc: 'Default license',
       required: false
     });
+
+    this.option('subDir', {
```

Comment:
instead of `subDir` can we name it `output` and give it default value `LICENSE`. This way you can specify it with the option.

subDir sounds very specific and no command line tools usually have it. `output`, `outputFile` or `outputDir`